Vagrantfile

```diff
@@ -148,6 +148,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                 # expose collins port to host for ease of management
                 node.vm.network "forwarded_port", guest: 9000, host: 9000
 
+                # expose UCP UI to the host on port 8080
```

Comment:
minor: this comment needs update as well, or better yet we can remove the port value from the comment so there is one less place to manage.